springCRUD/src/main/java/com/pironeer/springCRUD/service/ReplyService.java

```diff
@@ -0,0 +1,59 @@
+package com.pironeer.springCRUD.service;
+
+import com.pironeer.springCRUD.dto.request.ReplyCreateRequest;
+import com.pironeer.springCRUD.dto.request.ReplyUpdateRequest;
+import com.pironeer.springCRUD.dto.response.ReplyResponse;
+import com.pironeer.springCRUD.mapper.ReplyMapper;
+import com.pironeer.springCRUD.repository.CommentRepository;
+import com.pironeer.springCRUD.repository.ReplyRepository;
+import com.pironeer.springCRUD.repository.domain.Comment;
+import com.pironeer.springCRUD.repository.domain.Reply;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class ReplyService {
+    private final ReplyRepository replyRepository;
+    private final CommentRepository commentRepository;
+
+    // 대댓글 생성
+    public void save(Long commentId, ReplyCreateRequest request) {
+        Comment comment = commentRepository.findById(commentId)
+                .orElseThrow(() -> new RuntimeException("COMMENT NOT FOUND"));
+        Reply reply = ReplyMapper.from(request);
+        reply.setCommentId(commentId);
```

Comment:
이건 요렇게도 바꿀 수 있을 것 같아요! Setter는 가능하면 사용을 지양하는 게 좋아요
```suggestion
        Reply reply = ReplyMapper.from(request, commentId);
```